captalize_decorator.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'base-decorator'
+
+class CalitalizeDecorator < BaseDecorator
+  def initialize(nameable)
+    super()
+    @nameable = nameable
+  end
+
+  def correct_name
+    super.capitalize
+  end
```

Comment:
- [ ] For this section, you were supposed to "Implement a method correct_name that capitalizes the output of @nameable.correct_name.". This can be implemented like this: ```@nameable.correct_name.capitalize```